src/Mod/Surface/Gui/TaskFilling.cpp

```diff
@@ -569,7 +569,7 @@ void FillingPanel::on_listBoundary_itemDoubleClicked(QListWidgetItem* item)
 
 void FillingPanel::onSelectionChanged(const Gui::SelectionChanges& msg)
 {
-    if (selectionMode == None)
+    if (selectionMode is None)
```

Comment:
This is not Python code; this is C++. Therefore, the equals `==` cannot be changed to `is`. It breaks the code.

Reply:
:facepalm:  
Thanks for catching this, vocx-fc.  
I though i check before using `grep -lR '== None' | xargs sed -i 's/== None/is None/g'` but next time I'll use the grep `--exclude-dir=*.{cpp,h}` flag as well.